app/src/pages/inside/common/statusDropdown/statusDropdown.jsx

```diff
@@ -46,34 +49,41 @@ const messages = defineMessages({
 @connect(
   (state) => ({
     currentProject: activeProjectSelector(state),
+    currentPage: pageSelector(state),
   }),
   {
-    fetchFunc: fetchTestItemsAction,
+    fetchTestItems: fetchTestItemsAction,
```

Comment:
The same is about `fetchFunc`. It will be better to provide necessary function directly from parent component instead of choosing it here.